cnf-certification-test/platform/suite.go

```diff
@@ -114,8 +118,30 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 			ginkgo.Skip(" non ocp cluster ")
 		}
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestServiceMeshIdentifier)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.DebugPods)
+		TestServiceMesh(&env)
+	})
 })
 
+func TestServiceMesh(env *provider.TestEnvironment) {
+	var badPods []string
+	for _, put := range env.Pods {
+		data := put.Containers
+		fmt.Println(data)
+		for _, d := range data {
+			if d.Status.Name == istio {
+				tnf.ClaimFilePrintf("For pods %s ,ns %s we have service mesh", put.Data.Name, put.Data.Namespace)
+				continue
+			}
+			badPods = append(badPods, put.Data.Name)
+		}
+	}
+	logrus.Println("bad pods ", badPods)
```

Comment:
Should we add a line like: `gomega.Expect(badPods).To(gomega.BeNil())` to fail the test if `badPods` is populated?

Reply:
I think that you are right but Im not sure. this need to check for every pod if we have Istio , but in the pod have containers and can to be situation that part of containers no have Istio.

Reply:
No exactly. 
I was update the code, maybe now its better clear, I was save all the container of the pods that no have Istio, because can to be situation that in part of containers in specific pod have Istio and in others no have , so we can to save the pod and thats it but I think that maybe we can to save just the container of the pod that no have Istio. What do you think? @aabughosh

Reply:
u can do that but u also need to add as what brandon said the failure of the test

Reply:
I also think we should probably have a better log message here other than `bad pods` similar to #260.  We're no longer using gomega and a better log could be helpful for debugging.